### PR TITLE
Updating of the ERGA EAR tool

### PR DESCRIPTION
**Pull request by SaimMomin12**
This pull request updates the **ERGA EAR** tool as per the recent changes made by @diegomics

---

Changes and Updates:

- Added a new Methods section to specify used tools along with version and tools parameters
- Updated make_ear.py script (Linting + Reformatting)
- Updated test data 
- Version Bump

@bgruening could you please review this?

### CODE REVIEW

File: tools/ear/make_EAR.xml [hunk 1]
```diff
@@ -282,6 +298,26 @@ NOTES:
                     <param name="hap2_exists_curated" value="no"/>
                 </conditional>
             </section>
+            <section name="method_data">
```
Comment (bgruening) [inline tools/ear/make_EAR.xml:301]: Why is this twice here? Should it be a macro?

File: tools/ear/macros.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <macros>
     <token name="@TOOL_VERSION@">1.0.0</token>
```
[the remark this answers is not in the document]
Comment (SaimMomin12) [reply tools/ear/macros.xml:2]: But i realise, the version mentioned in EAR is merely a date

File: tools/ear/make_EAR.xml (hunk 1 above)
Comment (SaimMomin12) [reply tools/ear/make_EAR.xml:301]: There are two different parameters in the `methods_data` and are not repetitive

Comment (bgruening) [reply tools/ear/make_EAR.xml:301]: macros can be parameterized

File: tools/ear/macros.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (bgruening) [reply tools/ear/macros.xml:2]: I don't mind, it just easier for the user support if the galaxy version matches the tool version

File: tools/ear/make_EAR.xml (hunk 1 above)
Comment (SaimMomin12) [reply tools/ear/make_EAR.xml:301]: Okay, have made the change

File: tools/ear/macros.xml (hunk 2 above)
[the remark this answers is not in the document]
Comment (SaimMomin12) [reply tools/ear/macros.xml:2]: Sure, made the change

**Pull request merged**